app/views/register/FailedMatchView.scala.html

```diff
@@ -65,6 +65,6 @@
 
     <h2>@messages("failedMatch.h2")</h2>
 
-    <p>@messages("failedMatch.paragraph4") <a href="@config.helplineUrl" target="_blank" rel="external">@messages("failedMatch.contactLink")</a> @messages("failedMatch.paragraph5")</p>
+    <p>@messages("failedMatch.paragraph4") <a href="@config.helplineUrl" target="_blank" rel="external">@messages("failedMatch.contactLink")</a>@messages("failedMatch.paragraph5")</p>
```

Comment:
Should there not be a space between the link and the following paragraph text?

Reply:
the next paragraph starts with a fullstop so it was putting a space in before the fullstop.